pkgs/applications/audio/sfizz/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub , cmake, libjack2, libsndfile }:
+
+stdenv.mkDerivation rec {
+  pname = "sfizz";
+  version = "unstable-2020-01-20";
+
+  src = fetchFromGitHub {
+    owner = "sfztools";
+    repo = pname;
+    rev = "864fe57c54513cf0cbbcf43ab9efd97d9f2b8976";
+    sha256 = "0z74a1vag9iqqywkkvx3yg9l63as9f43lvjhal9xxmimn630mi3g";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ cmake libjack2 libsndfile ];
```

sorry didn't see before
```suggestion
  nativeBuildInputs = [ cmake ];

  buildInputs = [ libjack2 libsndfile ];
```